project1/css/index.css

```diff
@@ -1,64 +1,72 @@
-/* http://meyerweb.com/eric/tools/css/reset/ 
-   v2.0 | 20110126
-   License: none (public domain)
-*/
-
-html, body, div, span, applet, object, iframe,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+.container {
+	display: flex;
+	flex-direction: column;
+	border: 1px solid silver;
 }
-/* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+
+.header {
+	display: flex;
+	flex-direction: column;
+	border: 1px solid silver;
 }
-body {
-	line-height: 1;
+
+.navbar {
+	display: flex;
+	flex-direction: column;
 }
-ol, ul {
+
+.navButtons {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
 	list-style: none;
 }
-blockquote, q {
-	quotes: none;
+
+a {
+	text-decoration: none;
+	color: grey;
+	font-family: sans-serif;
+}
+
+.cta {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	border-bottom: 1px solid black;
+	padding-bottom: 10%;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+
+.cta-text {
+	display: flex;
+  flex-direction: column;
+  justify-content: center;
 }
-table {
-	border-collapse: collapse;
-	border-spacing: 0;
+
+.innovation {
+	font-family: bangers;
+	font-size: 450%;
 }
 
-/* Set every element's box-sizing to border-box */
-* {
-    box-sizing: border-box;
+.getStarted {
+	font-family: sans-serif;
+	background-color: white;
+	width: 14vw;
+	height: 7vh;
```

Comment:
I would use `px` here for the button size instead of `vh` and `vw`. Those are relative sizes, whereas `px` is a fixed size no matter what size the viewing device is.